src/app/auth/pages/connection-page/connection-page.component.html

```diff
@@ -39,10 +39,10 @@ <h1>{{ title }}</h1>
     <button type="submit" class="btn btn-info" [disabled]="loginForm.form.invalid">Se connecter</button>
   </form>
 
-  <p><a *appPermission="'admin'" id="testButton" class="btn btn-warning" [routerLink]="['/games']">Return to Games</a></p>
+  <p><a *displayIfIdentified="'admin'" id="returnButton" class="btn btn-warning" [routerLink]="['/games']">Return to Games</a></p>
 
-  <p><a *myNgIf="loggedIn" id="testButton2" class="btn btn-danger" [routerLink]="['/games']">Games</a></p>
+  <p><a *myNgIf="loggedIn" id="gamesButton" class="btn btn-danger" [routerLink]="['/games']">Games</a></p>
 
-  <app-message-box *ngIf="activeMsg" [class]="msgClass" [text]="msgText"></app-message-box>
+  <app-message-box *ngIf="activeMsg" [typeMessage]="typeMsg" [text]="msgText"></app-message-box>
```

Comment:
Qu'est ce que tu penses de quelque chose comme ça ?
```suggestion
  <app-message-box *ngIf="error" typeMessage="erreur" text="email ou mdp passe incorrect"></app-message-box>
```
avec cette approche tu as juste à setter un booléen `error` à true quand il y a une erreur. Et comme tu as seulement un seul message dans ton cas et un seul type de message tu n'as pas besoin de les variabiliser.

Reply:
Oui, j'y avais pensé mais je m'étais dit que c'était peut-être pas bien de passer directement des strings, par exemple si l'on voulait plus tard passer un autre type de message en fonction d'autres critères.